cmd/quadlet/main.go

```diff
@@ -59,6 +55,12 @@ var (
 	}
 )
 
+var (
+	UnitDirAdminUser         string
```

Comment:
Sorry for the nit. But, is there a reason why `UnitDirAdminUser` and `SystemUserDirLevel` are exposed and `resolvedUnitDirAdminUser` isn't? Are they used outside of the module?